### PR TITLE
Adding plotly examples app

### PR DESCRIPTION
**Pull request by srmukher**
Adding plotly examples app

### CODE REVIEW

File: apps/plotly_examples/src/data/parsed_data.json [hunk 1]
```diff
@@ -0,0 +1,4331 @@
+[
+  {
+    "data": [
+      {
+        "type": "scatter",
```
Comment (AtishayMsft) [inline apps/plotly_examples/src/data/parsed_data.json:5]: This file is very big. Could you move 1 case to 1 file for better readability and maintenance.

Comment (srmukher) [reply apps/plotly_examples/src/data/parsed_data.json:5]: so there will be 100 such files?

Comment (AtishayMsft) [reply apps/plotly_examples/src/data/parsed_data.json:5]: yes

Comment (AtishayMsft) [reply apps/plotly_examples/src/data/parsed_data.json:5]: You can take the comment as follow up.

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**